Bio/_utils.py

```diff
@@ -46,6 +46,22 @@ def read_forward(handle):
             return line
 
 
+def read_header(handle, length):
+    """Read the specified number of bytes from the given handle.
```

Comment:
Maybe bytes -> characters?

As written, this should work just fine on both raw handles (bytes) and text handles (unicode strings).